src/sentry/static/sentry/app/components/button.tsx

```diff
@@ -274,23 +275,30 @@ const getColors = ({priority, disabled, borderless, theme}: StyledButtonProps) =
 };
 
 const StyledButton = styled(
-  ({forwardRef, external, ...props}) => {
-    // Get component to use based on existence of `to` or `href` properties
-    // Can be react-router `Link`, `a`, or `button`
-    if (props.to) {
-      return <Link ref={forwardRef} {...props} />;
-    }
+  React.forwardRef<any, ButtonProps>(
+    ({forwardRef, size: _size, external, to, href, ...props}: Props, forwardRefAlt) => {
```

Comment:
Please, what does `Alt`  stand for? Alternative?

Reply:
Yeah more or less, it's the other forwarded ref

Reply:
should we rename it to something clearer? suggestions: `otherForwardRef` or `extraForwardRef` or `secondForwardRef`